packages/x-charts/README.md

```diff
@@ -0,0 +1,3 @@
+# @mui/x-charts
+
+TODO
```

Comment:
```suggestion
Package under development.
The alpha phase will start in the coming months, stay tuned !
```

In case somebody looks at it on Github